docs/team/marcuslowhuiyu.md

```diff
@@ -3,20 +3,25 @@ layout: page
 title: Marcus Low Hui Yu's Project Portfolio Page
 ---
 ## Overview of TutHub
-TutHub is a desktop app for tech-savvy tuition agents who want to keep track of hundreds of tutors’ profiles in a more organised and systematic way compared to excel sheets, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI).
+Tuthub is a desktop app for NUS professors who wish to track and choose their next batch of teaching assistants/tutors based on their past performance and records but have little time to spare for tedious administrative work. Tuthub helps profs who can type fast find the best TAs faster than traditional Graphical User Interface (GUI) apps.
 
 ## Summary of Contributions
 ### Code Contributed
-[RepoSense Link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=marcuslowhuiyu&breakdown=true)
+Around 500 lines of code according to the [RepoSense Link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=marcuslowhuiyu&breakdown=true)
 
 ### Enhancements implemented
-_to be added soon_
+- **New Feature**: Added `Comments` to `Tutor`
+    - Purpose: Allows comments to be added to each tutor
```

Comment:
Perhaps you could elaborate more on how it was implemented, or the possible use of comments